src/main/resources/plugin.yml

```diff
@@ -38,6 +38,7 @@ permissions:
         description: User is able to do all /siegewar commands.
         default: false
         children:
+            siegewar.command.siegewar.collect: true

```

Comment:
The collect node should probably also get its own entry in the plugin.yml with default set to true, so there is one less node needed to be given out manually.

Reply:
Fixed